code/go/0chain.net/smartcontract/storagesc/sc.go

```diff
@@ -207,6 +207,11 @@ func (sc *StorageSmartContract) Execute(t *transaction.Transaction,
 	case "cancel_allocation":
 		resp, err = sc.cancelAllocationRequest(t, input, balances)
 
+	case "add_curator":
+		resp, err = "", sc.addCurator(t, input, balances)
+	case "transfer_allocation_request":
```

Comment:
to drop _request suffix. or to add it to all the other cases.

Reply:
`finalizeAllocation`, `addBlobber`, `addValidator`, `stakePoolPayInterests`. The request suffix seems to be a minority choice. Also `cancel_allcoation` semes to not match `cancelAllocationRequest`.